.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+## Background
+<!-- What would a code reviewer, or a future dev, need to know about this PR in order to understand why this PR is necessary -->
+
+## Definition of Done
+
+- [ ] _Replace this line with individual "To-Do's" unique to your PR_
```

Comment:
What do you think of substituting `"To-Do's"` with `tasks finished`?

Reply:
Swapping "tasks" for "To-Do's" works for me!